resources/views/components/circle-button.blade.php

```diff
@@ -12,7 +12,7 @@
 @endphp
 
 <{{ $tag }} {{ $attributes->merge($defaultAttributes) }}>
-    <div @if($spinner) wire:loading.remove @endif>
+    <div @if($spinner) wire:target="{{ $spinner }}" wire:loading.remove @endif>
```

Comment:
Add a conditional wire target as the button component does

https://github.com/wireui/wireui/blob/main/resources/views/components/button.blade.php#L39